src/components/Tags.tsx

```diff
@@ -1,28 +1,41 @@
 import React from 'react'
 import { Tag, VerifiedIcon, CommunityIcon, BinanceIcon } from '@pancakeswap-libs/uikit'
+import useTheme from '../hooks/useTheme'
 
-const CoreTag = (props) => (
-  <Tag variant="secondary" outline startIcon={<VerifiedIcon />} {...props}>
-    Core
-  </Tag>
-)
+const CoreTag = (props) => {
+  const { isDark } = useTheme()
+  return (
+    <Tag variant="secondary" outline={!isDark} startIcon={<VerifiedIcon />} {...props}>
+      Core
+    </Tag>
```

Comment:
I think these are suppose to remain solid. If this is just to fix the icon color you can apply the color to the icon directly.

```
<Tag variant="secondary" outline startIcon={<VerifiedIcon color="secondary" />} {...props}>
```

Reply:
Done